docs/cli/cli.md

```diff
@@ -0,0 +1,122 @@
+# 💻 CLI for `jstz`
+
+This guide will instruct through how to use the command line interface for `jstz`.
+
+## Overview
+
+`jstz` provides a simple command line interface where the user can easily run the sandbox environment and test smart functions.
```

Comment:
Might be good to have a bullet point list with a list of top-level commands e.g. 
```suggestion
`jstz` offers a number of commands to manage and test your smart functions.
- [sandbox](...) - Locally deploy a sandbox environment.
- [bridge](...) - Interact with the XTZ asset bridge between Tezos and `jstz`.
- [deploy](...) - Deploy your smart function.
- [run](...) - Run your smart function.
- [repl](...) - Enter an interactive REPL for the `jstz` runtime.
...
```